Documents/idea document 2/ideadocument2.md

```diff
@@ -1,6 +1,9 @@
 # Introduction
+Nowadays a lot of (social) data is gathered and available on the internet via mediums like Facebook, Twitter, Instagram and many more. These platforms are widely used by people to share information to the public. Individual tweets, status posts and photos may not be that usefull for scientific studies, but when a multitude of these tweets, posts and photos are collected, characteristics can be concluded for certain groups of individuals that for example share the same interest, like sport activities. The first challenge here, is to collect a usefull amount of data and retrieve this latent characteristics that can identify the persons interest in any kind of sport activity. The difficulty is in analyzing, categorizing and grouping data the right way, to be able to extract and come to the right conclusions. Analyzing data can give insights on how people are communicating, what they are are interested about, what motivates and drives them and just general information on what they are doing. 
+Since data from the mentioned sources sometimes contain geographic references, it is even possible to identify and define characteristics for certain areas. Another challenge here is spatial analysis and clustering of the collected data.
 
 # Objective
+The goal of this project is to identify and characterise neighborhoods in Amsterdam based on sport activities, this requires the previously mentioned data collecting, categorization, and spatial clustering. The identified neighbourhoods can then be visualized on a map to get an overview of where different sport activities are popular, on what days of the week and on what time of the day. This can be usefull for event organizers, shopkeepers, or people that want to know where to find companions in there sport activities. 
```

Comment:
change project to project name?
this -> which
are popular, on what days of the week and on what time of the day -> are popular on which time of the day and what day of the week?
refrain using this
~~usefull~~